app/api/routers/health_check_router.py

```diff
@@ -1,17 +1,37 @@
 import logging
-from fastapi import APIRouter
+from typing import Callable
+from fastapi import APIRouter, Request, Response
+from fastapi.routing import APIRoute
+
 from api.models.health_check_status import HealthCheckStatus
 
+
 logger = logging.getLogger(__name__)
 
+
+class LogRoute(APIRoute):

```

Comment:
this must be moved to some central place. because this is something that will impact all routers

Reply:
moved to logger_config.py

Reply:
I dont see the move yet.

Reply:
Moved to loger_congig.py